### ISSUE TITLE
Install error on Raspberry

### ISSUE THREAD

**Issue by spawnrider** [smblott-github/chromix-too#1]
Hi,

I can't install Chromix-too on Raspberry Pi 2 (Debian 8.0 Jessie - Linux raspberrypi 4.1.7-v7+ #817 SMP PREEMPT Sat Sep 19 15:32:00 BST 2015 armv7l GNU/Linux).

There is an issue with NPM command line : 

```
pi@raspberrypi ~ $ sudo npm install -g chromix-too
npm WARN lifecycle chromix-too@0.0.5~preinstall: cannot run in wd %s %s (wd=%s) chromix-too@0.0.5 make build /root/.npm-global/lib/node_modules/.staging/chromix-too-630796e6
npm ERR! Linux 4.1.7-v7+
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "chromix-too"
npm ERR! node v0.10.29
npm ERR! npm  v3.10.7
npm ERR! path /root/.npm-global/lib/node_modules/chromix-too/server.js
npm ERR! code ENOENT
npm ERR! errno 34

npm ERR! enoent ENOENT, chmod '/root/.npm-global/lib/node_modules/chromix-too/server.js'
npm ERR! enoent ENOENT, chmod '/root/.npm-global/lib/node_modules/chromix-too/server.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /home/pi/npm-debug.log
```

Any idea ?

**Comment by smblott-github** [comment]
Hmm.  I just did the same on my own Raspberry Pi and the install was fine.  Then I tried my laptop (which worked previously) and I see the same error.  It seems to depend on the version of `npm`.

Looks like the `ENOENT` errors happen because the preinstall script failed.  The preinstall script just runs `make build` to convert the Coffeescript to Javascript.

[This comment](https://github.com/npm/npm/issues/3497#issuecomment-23966536) fixed the problem for me.  So, for you that would be:

```
sudo npm install --unsafe-perm -g chromix-too
```

I'll look into a more general solution.

Thanks for the heads up.

**Comment by smblott-github** [comment]
OK.  I got the problem, and it's fixed in `0.0.8`, now live.

I needed to have the JS packaged within the npm package, not build it on install.  And npm was ignoring the JS files because they were listed in `.gitignore`.  Solution was to create an empty `.npmignore`.

I'll close this now.  Shout if you continue to have problems.

**Comment by spawnrider** [comment]
Thanks for this quick fix :)
